include/deal.II/meshworker/scratch_data.h

```diff
@@ -0,0 +1,1301 @@
+
+    /**
+     * Initialize the internal FESubfaceValues to use the given @p subface_no,
+     * on @p face_no, on the given @p cell.
+     *
+     * If @p subface_no is numbers::invalid_unsigned_int, the reinit() function
+     * that takes only the @p cell and the @p face_no is called.
+     */
+    const FEFaceValuesBase<dim, spacedim> &
+    reinit(const typename DoFHandler<dim, spacedim>::active_cell_iterator &cell,
+           const unsigned int face_no,
+           const unsigned int subface_no);
+
+    /**
+     * Initialize the internal neighbor FEValues to use the given @p cell.
+     */
+    const FEValues<dim, spacedim> &
+    reinit_neighbor(
+      const typename DoFHandler<dim, spacedim>::active_cell_iterator &cell);
+
+    /**
+     * Initialize the internal FEFaceValues to use the given @p face_no on the given
+     * @p cell.
+     */
+    const FEFaceValues<dim, spacedim> &
+    reinit_neighbor(
+      const typename DoFHandler<dim, spacedim>::active_cell_iterator &cell,
+      const unsigned int                                              face_no);
+
+    /**
+     * Initialize the internal FESubfaceValues to use the given @p subface_no,
+     * on @p face_no, on the given @p cell.
+     *
+     * If @p subface_no is numbers::invalid_unsigned_int, the reinit() function
+     * that takes only the @p cell and the @p face_no is called.
+     */
+    const FEFaceValuesBase<dim, spacedim> &
+    reinit_neighbor(
+      const typename DoFHandler<dim, spacedim>::active_cell_iterator &cell,
+      const unsigned int                                              face_no,
+      const unsigned int subface_no);
+
+    /**
+     * Get the currently initialized FEValues.
+     *
+     * This function will return the internal FEValues if the
+     * reinit(cell) function was called last. If the reinit(cell, face_no)
+     * function was called, then this function returns the internal
+     * FEFaceValues, and if the reinit(cell, face_no, subface_no) function was
+     * called (with a valid @p subface_no argument), it returns the internal
+     * FESubfaceValues object.
+     */
+    const FEValuesBase<dim, spacedim> &
+    get_current_fe_values() const;
+
+    /**
+     * Get the currently initialized neighbor FEValues.
+     *
+     * This function will return the neighbor FEValues if the
+     * reinit_neighbor(cell) function was called last. If the
+     * reinit_neighbor(cell, face_no) function was called, then this function
+     * returns the internal neighbor FEFaceValues, and if the
+     * reinit_neighbor(cell, face_no, subface_no) function was
+     * called (with a valid @p subface_no argument), it returns the internal neighbor
+     * FESubfaceValues object.
+     */
+    const FEValuesBase<dim, spacedim> &
+    get_current_neighbor_fe_values() const;
+
+    /**
+     * Store internally a copy of the given object. The copied object is
+     * owned by this class, and is accessible via reference through the get()
```

Comment:
```suggestion
     * owned by this class, and is accessible via reference through the get_object_with_name()
```

Reply:
BTW. Sorry for not using the "suggest changes" utility earlier. I forgot about it :-/